packages/dds/merge-tree/api-report/merge-tree.legacy.alpha.api.md

```diff
@@ -538,15 +527,15 @@ export class Marker extends BaseSegment implements ReferencePosition, ISegment {
     // (undocumented)
     protected createSplitSegmentAt(pos: number): undefined;
     // (undocumented)
-    static fromJSONObject(spec: any): Marker | undefined;
+    static fromJSONObject(spec: IJSONSegment): Marker | undefined;
```

Comment:
Type-wise, this is a breaking change. Presumably this is the type the code was expecting? Assuming that, and assuming other kinds of input would error or be undefined behavior, this is probably fine. That said, we should definitely add a changeset to this PR that explicitly notes the incompatible type change with an explanation, just in case a consumer is dependent on the `any` typing here.

Reply:
Yep, this was the expected type, so I'll add a changeset. Any other kind of input to the method would return undefined, which would eventually throw an error to indicate the unexpected type.